core/src/main/scala/cats/data/Chain.scala

```diff
@@ -659,18 +679,18 @@ sealed abstract class Chain[+A] extends ChainCompat[A] {
 
   /**
    * Compares the length of this chain to a test value.
-   * 
+   *
    * The method does not call `length` directly; its running time
    * is `O(length min len)` instead of `O(length)`.
-   * 
+   *
    * @param  len the test value that gets compared with the length.
    * @return a negative value if `this.length < len`,
    *         zero if `this.length == len` or
    *         a positive value if `this.length > len`.
    * @note   an adapted version of
              [[https://github.com/scala/scala/blob/v2.13.8/src/library/scala/collection/Iterable.scala#L272-L288 Iterable#sizeCompare]]
              from Scala Library v2.13.8 is used in a part of the implementation.
-   * 
+   *
```

Comment:
Sorry, I completely missed this part. Seems that it is not derived from the Scala library anymore. It was so initially indeed, but then it was re-worked for the sake of optimization. Moreover, Cats has never been released with the original Scala-derived version. cc @DavidGregory084 @johnynek